### PR TITLE
fixes of urls (mostly repos)

### PR DESCRIPTION
**Pull request by Trilarion**

### CODE REVIEW

**Comment by pehsa** [comment]
Yesterday I created one PR for the URL fixes: #1123 
Theres some entry which will be conflicting with your changes. But you have more url fixes which I don't changed.

**Comment by Trilarion** [comment]
I just saw your PR after doing mine. I could solve the conflicts myself. Just first do your PR and then I adapt to it.

**Comment by Trilarion** [comment]
Rebased my branch on the current master here. From my side it's ready.

File: games/g.yaml [hunk 1]
```diff
@@ -427,14 +427,15 @@
   originals:
   - 'Grand Theft Auto: San Andreas'
   repo: https://github.com/grit-engine/grit-engine
+  url: https://github.com/grit-engine/grit-engine
```
Comment (cxong) [inline games/g.yaml:430]: this is the same as repo - don't need this
```suggestion
```

File: games/i.yaml [hunk 2]
```diff
@@ -97,10 +97,10 @@
   development: active
   originals:
   - 'Cube 2: Sauerbraten'
-  repo: https://github.com/inexor-game/code
+  repo: https://github.com/inexorgame/inexor-core
```
Comment (cxong) [inline games/i.yaml:100]:
```suggestion
  repo: https://github.com/inexorgame/vulkan-renderer
```
old repo is archived; this appears to be the active dev repo

**Review by cxong — APPROVED** [review]

**Pull request merged**